components/secretstores/secret_store.go

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2021 Layotto Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package secretstores
+
+// SecretStore is the interface for a component that handles secrets management.
+type SecretStore interface {
```

Comment:
Hi, can we reuse dapr's secret store components by importing them? So that we don't have to develop all of the secret components again....

Reply:
It can be done by introducing dapr's, but I don't think one component can be restricted. If there are similar products in the future, I hope users can expand through common interfaces.

Reply:
@MentosL Yes,one component can not fit all. 
But dapr already has an interface for secret components and many implementation,see https://github.com/dapr/components-contrib/blob/master/secretstores/secret_store.go
We can import these interface and components to make life easier :)

Reply:
我有表达清楚么 如果沟通不畅可以用中文 ：）

Reply:
Oh, I understand. I'll modify it on my end